### ISSUE TITLE
Honour system preference to minimize/zoom app on double click of title bar

### ISSUE THREAD

**Issue by gingerbeardman** [glushchenko/fsnotes#1594]
**Describe your feature request**
behaviour of the app on double click of title bar should use system preference

![Screen shot 2023-08-11 at 17 17 39](https://github.com/glushchenko/fsnotes/assets/49612/09d7f450-2d96-468e-8ab1-b789bc2a0ec7)

**Additional context**
from: https://github.com/glushchenko/fsnotes/issues/1591#issuecomment-1673237261

@eyelessjerry

**Comment by eyelessjerry** [comment]
That's what I think too, but it doesn't, hence my comment ... have not seen it in any other app until this one. Have never used the (now) default setting of zooming in the System Settings since the 80's ... .

**Comment by eyelessjerry** [comment]
<img width="457" alt="screen 2023-08-11 kl  21 09 16" src="https://github.com/glushchenko/fsnotes/assets/122881086/4bd0143e-9b90-4d63-b299-91febdcb8a74">

**Comment by gingerbeardman** [comment]
> That's what I think too 

I filed this bug for you! @eyelessjerry  One bug/request per issue

**Comment by glushchenko** [comment]
Finally i am here. Reimplemented. 

Thanks for the feedback, I didn't know about this feature.

**Comment by gingerbeardman** [comment]
Thanks!

I think it a very old macOS feature. I think the genesis was on System 7 where you could double click the title bar to fold the window up and only have the title bar on screen. Very cool!

**Comment by eyelessjerry** [comment]
Thanks, but does not seem to work in Version 6.9.1 (636) – maybe in the next version? Not sure when Apple added the feature as I think I used it maybe already in Classic OS and I did not notice how frequently I use it before installing FSNotes, which almost was a show stopper for me. Why use cmd+m or trying to find the yellow button, when it is easier clicking the window header, which unfortunately is made less useful in many apps these days with less space to click on – Mac users has always interacted with the header to move windows, rename the file, open the folder the file is located in, minimize windows in place (I think it was in Classic OS), etc. – it sort of essential to the Mac experience ... ;-). Just like you normally don't have to click in checkbox, but could as well click at the end of the same line in most settings ... .
